src/lib/kctl.py

```diff
@@ -88,3 +88,11 @@ def get_pods(self, namespace='default'):
     if not data["items"]:
       logging.debug(f"Kctl: no pods found in {namespace} namespace.")
     return data["items"]
+
+  def get_configmaps(self, namespace='default'):
```

Comment:
perhaps a test for this too?

Reply:
Done in https://github.com/artsy/opstools/pull/45/commits/e36d3569c1ecba047c4cf47b3974d06ff1f3b336